src/Rating.php

```diff
@@ -19,6 +20,7 @@ class Rating extends ClientBase
      *   The options to pass along to the Guzzle Client.
      *
      * @return \DOMDocument
+     * @throws \GuzzleHttp\Exception\GuzzleException
```

Comment:
This method does not seem to be throwing exceptions.

Reply:
`getRates()` calls `post()` which throws the exception... phpstorm gets angry if i remove it... should i ignore it?

Reply:
Ok PHPStorm wins. I always thought it would be tedious to follow the tree of all the functions that are called within a function and propagate `@throws`, but it seems that it's standard practice so that developers know and they can catch the exception. Keep as is, the same in the rest of the cases that I asked you to remove.